AudioConfigMCP49XX.h

```diff
@@ -0,0 +1,17 @@
+#ifndef AUDIOCONFIGMCP49XX_H
```

Comment:
I suggest naming this file AudioConfigExternalDAC, so that it will be easier to add further DACs in the future (without creating a separate file for each).